const e=JSON.parse('{"key":"v-b2edac52","path":"/interview/sourceCode/spring/Spring-tx.html","title":"Spring事务失效的8个场景","lang":"zh-CN","frontmatter":{"title":"Spring事务失效的8个场景","seo":false,"head":[]},"headers":[{"level":3,"title":"1. 抛出检查异常导致事务不能正确回滚","slug":"_1-抛出检查异常导致事务不能正确回滚","link":"#_1-抛出检查异常导致事务不能正确回滚","children":[]},{"level":3,"title":"2. 业务方法内自己 try-catch 异常导致事务不能正确回滚","slug":"_2-业务方法内自己-try-catch-异常导致事务不能正确回滚","link":"#_2-业务方法内自己-try-catch-异常导致事务不能正确回滚","children":[]},{"level":3,"title":"3. aop 切面顺序导致导致事务不能正确回滚","slug":"_3-aop-切面顺序导致导致事务不能正确回滚","link":"#_3-aop-切面顺序导致导致事务不能正确回滚","children":[]},{"level":3,"title":"4. 非 public 方法导致的事务失效","slug":"_4-非-public-方法导致的事务失效","link":"#_4-非-public-方法导致的事务失效","children":[]},{"level":3,"title":"5. 父子容器导致的事务失效","slug":"_5-父子容器导致的事务失效","link":"#_5-父子容器导致的事务失效","children":[]},{"level":3,"title":"6. 调用本类方法导致传播行为失效","slug":"_6-调用本类方法导致传播行为失效","link":"#_6-调用本类方法导致传播行为失效","children":[]},{"level":3,"title":"7. @Transactional 没有保证原子行为","slug":"_7-transactional-没有保证原子行为","link":"#_7-transactional-没有保证原子行为","children":[]},{"level":3,"title":"8. @Transactional 方法导致的 synchronized 失效","slug":"_8-transactional-方法导致的-synchronized-失效","link":"#_8-transactional-方法导致的-synchronized-失效","children":[]}],"git":{"createdTime":1695295148000,"updatedTime":1695295148000,"contributors":[{"name":"javaxiaobear","email":"2861184805@qq.com","commits":1}]},"readingTime":{"minutes":4.47,"words":1342},"filePathRelative":"interview/sourceCode/spring/Spring-tx.md","localizedDate":"2023年9月21日","copyright":{"author":"小熊学Java"},"excerpt":""}');export{e as data};
